_posts/2024-05-08-new-mailing-lists.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: "New Mailing Lists for Sandstorm"
+author: Jacob Weisz
+authorUrl: https://github.com/ocdtrekkie
+---
+
+Several of us have long talked about how to get away from Google Groups and when to do
+so. Google has not gotten better at managing spam both in terms of junk registrations
+and emails, as well as recently flagging one of our core community member's posts as
+spam more than once, despite them being here for several years. There's also the
+fundamental risk Google shutters or substantially alters Groups, and of course, it
+always seems a bit odd to ask users of a privacy-centered project to sign up with Google. 
+
+We've decided to launch new mailing lists using [groups.io][1], a service founded by the
+original developer of Yahoo! Groups before it was acquired. It's a solid platform with a
+very generous free tier and feature set. The service has been running since 2014 and the
+founder started making mailing list software in 1998, so the likelihood of stability and
+longevity here seems pretty high. It has some pleasant features on their site, but like
+any good traditional mailing list software, can handle pretty much everything you can
```

Comment:
I'd say "...it can handle..." and possibly ditch the preceding comma.

Also "everything you could want"